doc/developer-guides/hld/virtio-net.rst

```diff
@@ -509,6 +509,35 @@ is the virtual NIC created by acrn-dm:
       collisions:0 txqueuelen:1000
       RX bytes:0 (0.0 b) TX bytes:0 (0.0 b)
 
+How to Use MacVTap Interface
+============================
+In addition to Tap interface, ACRN also supports MacVTap interface. 
```

Comment:
change all spellings of Tap and tap to TAP (all caps).

Reply:
Fixed where necessary. Please check in the new PR.